src/main/java/com/example/StudentRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.example;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface StudentRepository extends JpaRepository<Student, Integer> {
+
+    List<Student> findByPaymentFalse();
```

Comment:
This method should be findByPaymen(boolean payment) and `false` should be passed as a parameter